### PR TITLE
Improve configurability BBE

### PR DESCRIPTION
**Pull request by HindujaB**
## Purpose
$subject

#Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/9319

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/configurable-variables/configurable_variables.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ For more information, see [Configure a sample Ballerina service](/learn/configur
 
 ::: code configurable_variables.bal :::
 
-::: out configurable_variables.out :::
\ No newline at end of file
+::: out configurable_variables.out :::
```
Comment (ravinperera00) [inline examples/configurable-variables/configurable_variables.md:9]: Nothing was changed here right? Or is this a bug?

Comment (HindujaB) [reply examples/configurable-variables/configurable_variables.md:9]: yeah I reverted the changes I did

**Review by ravinperera00 — APPROVED** [review]
LGTM

**Review by MaryamZi — APPROVED** [review]

**Comment by MaryamZi** [comment]
Need to send the changes to the 10.x branch also.

**Pull request merged**